### ISSUE TITLE
Add ROC curves to classifier accuracy visualization

### ISSUE THREAD

**Issue by nbokulich** [qiime2/q2-sample-classifier#157]
**Addition Description**
Add ROC curves to classifier accuracy visualization

**Current Behavior**
Only confusion matrix and accuracy scores are output

**Proposed Behavior**
Add ROC curves and AUC to the viz. Use `Probabilities` artifact/scores to calculate ROC/AUC. 
1. The `confusion-matrix` visualizer should accept `SampleData[Probabilities]` as an optional input.
2. The `classify-samples` pipeline should pipe Probabilities to `confusion-matrix` to generate this viz by default.

**References**
1. [roc_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html#sklearn.metrics.roc_curve)
2. [roc binary example](https://scikit-learn.org/stable/auto_examples/model_selection/plot_roc_crossval.html#sphx-glr-auto-examples-model-selection-plot-roc-crossval-py)
3. [roc multiclass example](https://scikit-learn.org/stable/auto_examples/model_selection/plot_roc.html)
4. [roc_auc_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score)
